src/samplers/number.js

```diff
@@ -1,31 +1,30 @@
-const faker = require('faker');
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 export function sampleNumber(schema) {
-  let res;
+  const MAX_VALUE = schema.type && schema.type === 'integer' ?
+    Number.MAX_SAFE_INTEGER : Number.MAX_VALUE;
+  const MIN_VALUE = Number.MIN_SAFE_INTEGER;
```

Comment:
Why do you no handle min of  `number` type?

Reply:
To support negative numbers based on some unit tests

Reply:
You did not answer the question. `Number.MIN_SAFE_INTEGER` is min for `integer`. `-Number.MAX_VALUE` is min for `number`.

Reply:
oh i tried to use MIN_VALUE but it was not registered in if as negative and did not think of using -MAX_VALUE

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_VALUE